assignments/function-conversion.js

```diff
@@ -4,27 +4,48 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+let myFunction = () => {
+    return("Function was invoked!")
+}
+console.log(myFunction());
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+let anotherFunction = (param) => {
+    return param;
+};
+console.log(anotherFunction("Example"));
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+let add = (param1, param2) => {
+    return param1 + param2;
+};
+console.log(add(1,2));
 
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
-
+let subtract = (param1, param2) => {
+    return param1 - param2;
+}
+console.log(subtract(1,2));
 
 // Stretch
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map((num) => {
+    return num * 3;
+})
+console.log(triple);
```

Comment:
nice job on these! you can technically get rid of the braces if you keep it on one line, and then don't need a return, but I prefer to do it your way always just so I'm set up to make a more complex function whenever